### PR TITLE
Load static server configuration from connections.yaml

### PR DESCRIPTION
**Pull request by david415**

### CODE REVIEW

File: src/allmydata/client.py [hunk 1]
```diff
@@ -360,32 +380,15 @@ def init_client_storage_broker(self):
...
-        if self.get_config("client-server-selection", "use_introducer",
-                           default=True, boolean=True):
-            sb.use_introducer(self.introducer_client)
+        # utilize the loaded static server specifications
+        servers = self.connections_config['servers']
+        for server_id in servers.keys():
+            if self.testing:
+                self.storage_broker.got_static_announcement(servers[server_id]['key_s'], servers[server_id]['announcement'])
+            else:
+                eventually(self.storage_broker.got_static_announcement, servers[server_id]['key_s'], servers[server_id]['announcement'])
```
Comment (daira) [inline src/allmydata/client.py]: This looks questionable.

**Comment by daira** [comment]
Apart from ``self.testing``, this looks great.

**Comment by daira** [comment]
ACK.

**Comment by david415** [comment]
why does it say the branch is non-mergeable?

**Comment by meejah** [comment]
LGTM +1

**Comment by daira** [comment]
LGTM.

**Comment by daira** [comment]
Rebased and pushed.

**Pull request closed without merging**